kafka/src/main/java/org/axonframework/extensions/kafka/eventhandling/consumer/subscribable/SubscribableKafkaMessageSource.java

```diff
@@ -75,7 +76,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(SubscribableKafkaMessageSource.class);
 
-    private final List<String> topics;
+    private final SubscriberBuilder subscriber;
```

Comment:
I am missing a test addition for this change.

Reply:
Latest push includes testing in the `SubscribableKafkaMessageSourceTest` for both "list" and "pattern" based subscriptions.